kitsune/sumo/static/sumo/js/questions.js

```diff
@@ -152,10 +152,6 @@
       .closest('ul')
       .toggleClass('show-details');
     });
-
-    if (!aaq.isDesktopFF() && !aaq.isMobileFF() && !aaq.isFirefoxOS()) {
```

Comment:
don't we need this section anymore (without the `isFirefoxOS` of course)?

Reply:
We've moved it into the above file: https://github.com/mozilla/kitsune/pull/4293/files#diff-1fb1ffccd814794452284c9186f28d79R44-R46